Description: Raise an exception if the user attempts to use the
 functionality which was supported by the lzhuf binary.
 The binary is removed due to lack of compliance with the dfsg
 and is generally only useful to windows users
Author: Steve Conklin <steve@conklinhouse.com>
Last-Update: 2011-02-18

--- a/d_rats/wl2k.py
+++ b/d_rats/wl2k.py
@@ -39,6 +39,7 @@
     return string.replace("\n", r"\n").replace("\r", r"\r")
 
 def run_lzhuf(cmd, data):
+    raise Exception("Not supported due to lack of license for lzhuf")
     p = platform.get_platform()
 
     cwd = tempfile.mkdtemp()
